### PR TITLE
Activate news and member factories

### PR DESCRIPTION
**Pull request by Joas1988**

### CODE REVIEW

File: spec/factories/project.rb [hunk 1]
```diff
@@ -17,6 +18,7 @@
       project.project_time_entry_activities = FactoryBot.create_list(:time_entry_activity, 1) if project.project_time_entry_activities.empty?
 
       FactoryBot.create_list :issue, evaluator.number_of_issues, project: project
+      FactoryGirl.create_list :member, evaluator.number_of_members, :project => project, :roles => [FactoryGirl.create(:role)]
```
Comment (luk4s) [inline spec/factories/project.rb]: Do not combine hash styles. Use prefer 1.9
https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashsyntax

File: spec/factories/members.rb [hunk 2]
```diff
@@ -1,4 +1,3 @@
-=begin
 FactoryBot.define do
```
Comment (luk4s) [inline spec/factories/members.rb]: Missing test for this Factory 
(spec/factory_spec.rb)

File: spec/factories/news.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+
+  factory :news do
```
Comment (luk4s) [inline spec/factories/news.rb:3]: missing test for this Factory
(spec/factory_spec.rb)

**Review by luk4s — CHANGES_REQUESTED** [review]
* Please add new factories to tests. 
* Please follow code style used in ryspec

File: spec/factories/project.rb [hunk 4]
```diff
@@ -17,6 +18,9 @@
       project.project_time_entry_activities = FactoryBot.create_list(:time_entry_activity, 1) if project.project_time_entry_activities.empty?
 
       FactoryBot.create_list :issue, evaluator.number_of_issues, project: project
+      evaluator.number_of_members.times do
+        project.add_default_member(user)
```
Comment (luk4s) [inline spec/factories/project.rb]: Missing test for this line.

**Review by luk4s — CHANGES_REQUESTED** [review]
I think that this will not work. Because `user` is not defined on project

**Comment by Joas1988** [comment]
Member changes reverted, Redmine logic for creating new projects counts on setting environment that is not currently modeled by factories, so It's better to solve explicitly in specs.

**Review by luk4s — APPROVED** [review]

**Pull request merged**